tap_azure_git/__init__.py

```diff
@@ -33,10 +35,15 @@
     'refs': ['id'],
     'commit_files': ['id'],
     'repositories': ['id'],
-    'annotated_tags': ['id']
+    'annotated_tags': ['id'],
+    'builds': ['_sdc_id'],
```

Comment:
These items have an "id" field already.  This is a unique identifier that gets built for these items.